### ISSUE TITLE
v. 4.6 not compatible with Windows 7

### ISSUE THREAD

**Issue by apprenti-sorcier** [darktable-org/darktable#16765]
### Describe the bug

v. 4.4.2 and previous are compatible with Windows 7.
v. 4.6 and 4.6.1 are not, but there is no information about it in the change log. See the error message.
![Image du presse-papiers](https://github.com/darktable-org/darktable/assets/161721020/8a91a4a5-c42d-43d9-8c1a-05e4e0938540)

### Steps to reproduce

Install 4.6.0 or 4.6.1 on Windows 7. No problem with the installation.
Try to run. Get the error message.

### Where did you obtain darktable from?

downloaded from www.darktable.org

### darktable version

4.6

### What OS are you using?

Windows

### What is the version of your OS?

Windows 7

### Are you using OpenCL GPU in darktable?

None

**Comment by pehar1** [comment]
Windows 7 has been obsolete for years. I don’t think anyone is willing or able to continue supporting this outdated operating system.
Incidentally, it can be extremely dangerous to use an operating system for which there have been no security updates for years.

**Comment by apprenti-sorcier** [comment]
Then the change log should state it so that users don’t download incompatible versions. The installer should not install an incompatible version.

**Comment by victoryforce** [comment]
> Then the change log should state it so that users don’t download incompatible versions.

This suggestion is no better than informing about the minimum required version of Windows in the README file. At least the README is displayed immediately when opening the darktable GitHub repository. As for me, the changelog (or its variations, such as the release notes of the latest version, etc.) is the last thing I read when evaluating whether it is worth trying to install unfamiliar software.

> The installer should not install an incompatible version.

Yes, you are right here. It's a legitimate wish. While I won't be reopening your issue, I've put this item on my list as an opportunity for improvement. But _it will have the lowest possible priority_. Why exactly?

Currently, among the contributors there is no one who would be a Windows developer in the full sense. I think that adding the ability to distinguish Windows versions to the installer (we use NSIS) is definitely a doable task. But a non-Windows developer will have to spend some time researching an unfamiliar tool. While the benefit of this is minimal: to save a few minutes of time for the really tiny number of users who will try to install darktable on Windows 7.

In summary, if you can contribute the necessary code, it will be gratefully accepted and integrated into darktable. Otherwise, don't expect a high likelihood that anyone will devote resources to this improvement.
